common/pb/admin_commands.proto

```diff
@@ -26,3 +27,13 @@ message Command_ReloadConfig {
 		optional Command_ReloadConfig ext = 1002;
 	}
 }
+
+message Command_AdjustMod {
+	extend AdminCommand {
+		optional Command_AdjustMod ext = 1003;
+	}
+	optional string admin_name = 1;
+	optional string user_name = 2;
+	optional bool should_be_mod = 3;
```

Comment:
These fields shouldn't be optional, the command doesn't make sense without them.